src/main/java/me/juancarloscp52/entropy/events/db/DVDEvent.java

```diff
@@ -18,15 +18,19 @@
 package me.juancarloscp52.entropy.events.db;
 
 import me.juancarloscp52.entropy.events.AbstractTimedEvent;
+import me.juancarloscp52.entropy.events.EventCategory;
+import me.juancarloscp52.entropy.events.EventType;
 import net.minecraft.client.DeltaTracker;
 import net.minecraft.client.Minecraft;
 import net.minecraft.client.gui.GuiGraphics;
 import net.minecraft.util.ARGB;
 import net.minecraft.util.Mth;
+
 import java.util.Random;
 
 public class DVDEvent extends AbstractTimedEvent {
 
+    public static final EventType<DVDEvent> TYPE = EventType.builder(DVDEvent::new).category(EventCategory.SCREEN_ASPECT).build();
```

Comment:
Change in category here? That change makes sense to me, but wanted to make sure it's on purpose.

Reply:
It's on purpose, as it's the only event that had that type which is now no longer the case being properly categorized.